_news/107.md

```diff
@@ -34,21 +34,21 @@ You can also find the source code at [github.com](http://github.com) in the offi
 
 The Windows binary distributions can be downloaded from
 
-<https://www.erlang.org/download/otp_win32_19.1.exe>
+<https://erlang.org/download/otp_win32_19.1.exe>
 
-<https://www.erlang.org/download/otp_win64_19.1.exe>
+<https://erlang.org/download/otp_win64_19.1.exe>
 
 You can also download the complete HTML documentation or the Unix manual files
 
-<https://www.erlang.org/download/otp_doc_html_19.1.tar.gz>
-<https://www.erlang.org/download/otp_doc_man_19.1.tar.gz>
+<https://erlang.org/download/otp_doc_html_19.1.tar.gz>
+<https://erlang.org/download/otp_doc_man_19.1.tar.gz>
 
 
  You can also read the documentation on-line here:
  (see the Release Notes mentioned above for release notes which
  are not updated in the doc, but the new functionality is)
 
-[https://www.erlang.org/doc/](/doc/)
```

Comment:
incorrect

Reply:
the display link you still be https://www.erlang.org/... ?

Reply:
yes, exactly. The link should point to `/some/thing/`, but what is displayed should be the full path.

Reply:
Fixed.